src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Net46.cs

```diff
@@ -0,0 +1,1193 @@
+            }
+
+            set
+            {
+                // Setting the value to 0 is OK: It means the user doesn't want the handler to buffer content.
+                if (value < 0)
+                {
+                    throw new ArgumentOutOfRangeException("value");
+                }
+
+                if (value > HttpContent.MaxBufferSize)
+                {
+                    throw new ArgumentOutOfRangeException("value", value,
+                        string.Format(CultureInfo.InvariantCulture, SR.net_http_content_buffersize_limit,
+                        HttpContent.MaxBufferSize));
+                }
+
+                CheckDisposedOrStarted();
+                _maxRequestContentBufferSize = value;
+            }
+        }
+
+        public int MaxResponseHeadersLength
+        {
+            get
+            {
+                return _maxResponseHeadersLength;
+            }
+
+            set
+            {
+                if (value <= 0)
+                {
+                    throw new ArgumentOutOfRangeException("value");
+                }
+
+                CheckDisposedOrStarted();
+                _maxResponseHeadersLength = value;
+            }            
+        }
+
+        public bool PreAuthenticate
+        {
+            get
+            {
+                return _preAuthenticate;
+            }
+
+            set
+            {
+                CheckDisposedOrStarted();
+                _preAuthenticate = value;
+            }
+        }
+
+        public IDictionary<String, Object> Properties
+        {
+            get
+            {
+                if (_properties == null)
+                {
+                    _properties = new Dictionary<String, object>();
+                }
+
+                return _properties;
+            }
+        }
+
+        public IWebProxy Proxy
+        {
+            get
+            {
+                return _proxy;
+            }
+
+            [SecuritySafeCritical]
+            set
+            {
+                if (!UseProxy && value != null)
+                {
+                    throw new InvalidOperationException(String.Format(CultureInfo.InvariantCulture,
+                        SR.net_http_invalid_enable_first, "UseProxy", "true"));
+                }
+                CheckDisposedOrStarted();
+                ExceptionHelper.WebPermissionUnrestricted.Demand();
+                _proxy = value;
+            }
+        }
+
+        public Func<HttpRequestMessage, X509Certificate2, X509Chain, SslPolicyErrors, bool> ServerCertificateCustomValidationCallback
+        {
+            get
+            {
+                return _serverCertificateCustomValidationCallback;
+            }
+
+            set
+            {
+                CheckDisposedOrStarted();
+                _serverCertificateCustomValidationCallback = value;
+            }
+        }
+
+        public SslProtocols SslProtocols
+        {
+            // TODO: Issue #11100.
```

Comment:
@karelz @davidsh So we don't forget, we may want to create a list and track them in the issue.